src/Sarif.FunctionalTests/SarifConverterTests.cs

```diff
@@ -77,6 +78,32 @@ public void PREfastConverter_EndToEnd()
             BatchRunConverter(ToolFormat.PREfast);
         }
 
+        [Fact]
+        public void ValidateDeserializationAndPrettifiedSarifStringComparisionLogic()
+        {
+            string sarifComparatorFileFolder = "TestSarifComparator/Prettified";            
+            string testDirectory = Path.Combine(TestDirectory, sarifComparatorFileFolder);
+            string generatedSarif = File.ReadAllText(Directory.GetFiles(testDirectory, "generated.sarif")[0]);
+            string expectedSarif = File.ReadAllText(Directory.GetFiles(testDirectory, "expected.sarif")[0]);
+            CanGeneratedSarifBeDeserializedToExpectedSarif(generatedSarif, expectedSarif).Should().Be(true);
+        }
+
+        [Fact]
+        public void ValidateDeserializationAndNotPrettifiedSarifStringComparisionLogic()
+        {
+            string sarifComparatorFileFolder = "TestSarifComparator/NotPrettified";
+            string testDirectory = Path.Combine(TestDirectory, sarifComparatorFileFolder);
+            string generatedSarif = File.ReadAllText(Directory.GetFiles(testDirectory, "generated.sarif")[0]);
+            string expectedSarif = File.ReadAllText(Directory.GetFiles(testDirectory, "expected.sarif")[0]);
+            CanGeneratedSarifBeDeserializedToExpectedSarif(generatedSarif, expectedSarif).Should().Be(true);
+        }
+
+        [Fact]
```

Comment:
Remove this empty test.

Reply:
Removed the tests and manually validated the changes.